test/unit/api_spec.js

```diff
@@ -12,7 +12,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+import { buildGetDocumentParams, NodeFileReaderFactory, TEST_PDFS_PATH } from './test_utils';
```

Comment:
This line is too long, it should be:
```javascript

import {
  buildGetDocumentParams, NodeFileReaderFactory, TEST_PDFS_PATH
} from './test_utils';
```

And please keep *one* blank line between the copyright header and the first `import`.